shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/LedgerState.hs

```diff
@@ -370,7 +370,7 @@ instance Era era => FromCBOR (DPState era) where
 
 data RewardUpdate era = RewardUpdate
   { deltaT :: !Coin,
-    deltaR :: !Coin,
```

Comment:
make all three Deltas

Reply:
Ha, we just had the same idea: https://github.com/input-output-hk/ouroboros-network/pull/2702#discussion_r510225313

Reply:
next PR! (if it wasnt done already)